parm/atm/hofx/hofx4d.yaml.j2

```diff
@@ -37,8 +37,12 @@ model:
   filename_sfcd: '%yyyy%mm%dd.%hh%MM%ss.sfc_data.nc'
   filename_sfcw: '%yyyy%mm%dd.%hh%MM%ss.fv_srf_wnd.res.nc'
   filename_cplr: '%yyyy%mm%dd.%hh%MM%ss.coupler.res'
-  tstep: $(BKG_TSTEP)
+  tstep: '{{ BKG_TSTEP }}'
   model variables: [ua,va,t,delp,sphum,ice_wat,liq_wat,o3mr,phis,
                     slmsk,sheleg,tsea,vtype,stype,vfrac,stc,smc,snwdph,
                     u_srf,v_srf,f10m]
-observations: !INC ${OBS_LIST}
+observations:
+{% filter indent(width=4) %}
```

Comment:
```suggestion
{% filter indent(width=2) %}
```

Reply:
See comment above

Reply:
Indentation for `parm/atm/hofx` templates corrected at b3c8737.